flatnotes/umbrel-app.yml

```
@@ -0,0 +1,38 @@
+manifestVersion: 1
+id: flatnotes
+category: files
+name: flatnotes
+version: "5.5.4"
+tagline: A self-hosted, database-less markdown note-taking app
+description: >
+  📝 Flatnotes is a clean, self-hosted note-taking app that treats your ideas with respect. Every note you write lives as a plain Markdown file in a simple folder on your own server. 
+  No databases lurking in the background, no vendor lock-in, no complicated structure. You remain in full control of your thoughts and your data at all times.
+
+
+
+  The interface keeps distractions out of the way so you can focus entirely on writing. Quickly create notes, add tags, and jump across your knowledge with powerful full-text search.
+  Wikilinks make it effortless to connect ideas, turning your notes into a personal knowledge network. Switch between raw Markdown and a polished WYSIWYG editor whenever your mood shifts.
+
+
+
+  Flatnotes stays lightweight yet capable. Everything syncs instantly, even if you edit files outside the app.
+  Browse from any device with a responsive design that feels right whether you are at your desk or out in the world. 
+  Theming keeps your space cozy with beautiful light and dark modes.
+
+
+
+  Flatnotes frees your notes from proprietary cages and lets them breathe as future-proof Markdown. 
+  It can serve as your personal journal, research companion, digital garden, or anything else your creativity grows into. Pure writing. Pure ownership. Zero noise.
+developer: Adam Dullage
+website: https://github.com/dullage/flatnotes
+dependencies: []
+repo: https://github.com/dullage/flatnotes
+support: https://github.com/dullage/flatnotes/issues
+port: 8523
+gallery: []
+path: ""
+defaultUsername: "admin"
+defaultPassword: "umbrel"
```

Use of the generated password here.
```suggestion
deterministicPassword: true
```